### PR TITLE
Vizier Change

### PR DESCRIPTION
**Pull request by Merek2**
## About The Pull Request

Right. I will explain the changes here. I took a look at the Cleric skill list and realized that after Viziers had magic taken away and were made a healer class mostly, they still lack one important thing. The ability to be a mercenary. Sure a healer is nice to hire but why would any mercenary not have combat ability?

Basically, the other two subclass do have their own specialty that make them useful in combat. As we took away magic, I thought I would look at the comparison to cleric. NORMALLY I would talk about taking away skills from Adventurers but some of them are like that because they're the only non-church option etc. With that in mind I also see a lot less people actually wanting to play Vizier ever which was also my goto for a little.

Namely, the changes I make are to put them slightly on par with cleric. They have a cleric's ability to climb, a bit of combat ability for the sword which they get. I didn't put anything to expert as even cleric doesn't get that and it's a bit much for a non-templar role. This also still makes them weaker to a cleric in some ways, so basically what I did was make them work as an actual mercenary class. They used to have Journeyman sword and while I understand balancing is important, taking away their magic was enough, they should still work as a mercenary class.

Basically, if the idea is to have a healing merc, we want them to be able to do the mercenary thing. And healing while pretty useful is still not the same utility as magic, there needs to be compensation. None of these changes will put them on par with most fighting adventurers/town folk. It will make them a bit more viable though.

### CODE REVIEW

**Comment by Merek2** [comment]
And hilariously, they are still outperformed by the other two classes with sheer combat ability.

**Comment by chemistrymain3** [comment]
They get an extra 3 speed and a shamshir with decent skills as well as the ability to use crossbows. They don't need anything else.

**Comment by bearrrrrrrr** [comment]
"We're done with +1 -1 balance PR's."


The 3rd +1 -1 Balance PR of the week walking into the room:

**Comment by Lutowski** [comment]
we've a freeze on these kinds of PRs to encourage new content instead of number juggling, sorry but not what I'm looking for rn
